internal/logger/logger_test.go

```diff
@@ -0,0 +1,47 @@
+package logger
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestDefault(t *testing.T) {
+	tests := []struct {
+		name string
+		want *Logger
+	}{
+		{
+			name: "test_default",
+			want: Default(),
```

Comment:
 Может быть имеет смысл проверять что `Default(...)`  возвращает `slog.Default()`?  В противном случае проверяем что Default это Default.
```suggestion
			want: slog.Default(),
```